contracts/ReceivableStorage.sol

```diff
@@ -4,6 +4,8 @@ pragma solidity 0.8.19;
 import { Receivable } from "./libraries/types/DataTypes.sol";
 
 contract ReceivableStorage {
+    address public isleGlobal;
```

Comment:
Since this is the proxy address of isleGlobal, it won't change again unless we redeploy all the contract set. I suggest to mark this immutable.